### PR TITLE
Adds a Dorf religion.

### PR DESCRIPTION
**Pull request by DrSnips**
<u>A PR for DeadPoolJ.</u>
Adds a dorf religion datum.
:cl:
 * rscadd: Added a Dorf religion for chaplain.

### CODE REVIEW

**Comment by IzayaChan** [comment]
https://www.youtube.com/watch?v=KjHclWPVij0

File: code/datums/religions.dm [hunk 1]
```diff
@@ -880,3 +880,11 @@
 	//Add veganism disability
 	H.dna.SetSEState(VEGANBLOCK, 1)
 	domutcheck(H, null, 1)
+
+/datum/religion/dorf
+	name = "Dorfism"
+	deity_name = "Armok, God of Blood"
+	bible_name = list("How to Play Dwarf Fortress", "Book of Grudges", "Strike the Earth", "Lazy Newb Pack", "The Will of Armok", "Mining 101", "Hidden Fun Stuff and You")
```
Comment (DamianX) [inline code/datums/religions.dm]: shouldn't this be `bible_names`?

Comment (ShiftyRail) [reply code/datums/religions.dm]: Yes

**Comment by SirSkeleto** [comment]
eyy my suggestion got in

**Comment by jknpj** [comment]
I wonder if giving the chaplain dwarfism would be too much

**Pull request merged**